builder/chroot/step_check_root_device.go

```diff
@@ -6,20 +6,23 @@ import (
 
 	"github.com/hashicorp/packer-plugin-sdk/multistep"
 	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
-	oscgo "github.com/outscale/osc-sdk-go/v2"
+	oscgo "github.com/outscale/osc-sdk-go/v3/pkg/osc"
 )
 
 // StepCheckRootDevice makes sure the root device on the OMI is BSU-backed.
 type StepCheckRootDevice struct{}
 
-func (s *StepCheckRootDevice) Run(_ context.Context, state multistep.StateBag) multistep.StepAction {
+func (s *StepCheckRootDevice) Run(
+	_ context.Context,
+	state multistep.StateBag,
+) multistep.StepAction {
 	image := state.Get("source_image").(oscgo.Image)
 	ui := state.Get("ui").(packersdk.Ui)
 
 	ui.Say("Checking the root device on source OMI...")
 
 	// It must be BSU-backed otherwise the build won't work
-	if image.GetRootDeviceType() != "ebs" {
+	if image.RootDeviceType != "ebs" {
```

Comment:
This looks strange, as the API doc specifies:
```
// RootDeviceType The type of root device used by the OMI (always `bsu`).
```
It looks like it will always fail